progress-planner.php

```diff
@@ -26,6 +26,10 @@
 define( 'PROGRESS_PLANNER_DIR', __DIR__ );
 define( 'PROGRESS_PLANNER_URL', untrailingslashit( plugin_dir_url( __FILE__ ) ) );
 
+if ( ! defined( 'PROGRESS_PLANNER_REMOTE_SERVER_ROOT_URL' ) ) {
+	define( 'PROGRESS_PLANNER_REMOTE_SERVER_ROOT_URL', 'https://progressplanner.com' );
+}
```

Comment:
Instead of doing this, could we do it in the `Base` class, so we can call `\Progress_Planner\Base::REMOTE_SERVER_ROOT_URL`? I would prefer it as opposed to a global const...

Reply:
Or even a method so we can call `\progress_planner()->get_remote_server_root_url()`, and we can add a const check in there...
so something like
```php
public function get_remote_server_root_url() {
  return defined( 'PROGRESS_PLANNER_REMOTE_SERVER_ROOT_URL' )
    ? PROGRESS_PLANNER_REMOTE_SERVER_ROOT_URL
    : 'https://progressplanner.com';
}

Reply:
We can, the only important is that is defined in one place and that we can override it

Reply:
@aristath , updated

`classes/widgets/class-latest-badge.php` needed to be changed, since we can't concatenate strings in a property declaration.